.eslintrc.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+
+module.exports = {
+  parser: 'babel-eslint',
+  plugins: [
+    'react',
+    'react-hooks',
+    'flowtype',
+    'jest'
+  ],
+  extends: [
+    'standard',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:flowtype/recommended'
+  ],
+  env: {
+    es6: true,
+    node: true,
+    browser: true,
+    'jest/globals': true
+  },
+  settings: {
+    flowtype: {
+      onlyFilesWithFlowAnnotation: true
+    }
+  },
+  ignorePatterns: [
+    '**/flow-typed/',
+    '**/reports/',
+    '**/node_modules/',
+    '**/ios/main.jsbundle',
+    '**/stylelint.config.js',
+    '**/dist/',
+    '**/lib-dist/'
+  ],
+  rules: {
+    'arrow-parens': ['error', 'as-needed'],
+    curly: ['error', 'all'],
+    'jsx-quotes': ['error', 'prefer-single'],
+    'max-len': ['warn', { code: 120 }],
+    'no-loop-func': 'error',
+    'prefer-const': 'error',
+    'prefer-object-spread': 'error',
+    'prefer-template': 'error',
+
+    'react/display-name': 'off',
+    'react/no-access-state-in-setstate': 'error',
+    'react/no-did-mount-set-state': 'warn',
+    'react/no-did-update-set-state': 'warn',
+    'react/no-unescaped-entities': 'off',
+    'react/no-unused-prop-types': 'warn',
+    'react/no-typos': 'error',
+    'react/prefer-es6-class': ['error', 'always'],
+
+    'flowtype/delimiter-dangle': ['error', 'never'],
+    'flowtype/newline-after-flow-annotation': ['error', 'always'],
+    'flowtype/no-primitive-constructor-types': 'error',
+    'flowtype/no-weak-types': 'warn',
+    'flowtype/object-type-delimiter': ['error', 'comma'],
+    'flowtype/require-exact-type': ['warn', 'always'],
```

Comment:
Added this rule. It is responsible for most warnings. I think we should gradually fix all inexact types. Will close https://issues.tuerantuer.org/browse/LINT-8